files/Aboot/boot0.j2

```diff
@@ -395,7 +395,8 @@ extract_image() {
 
 extract_image_secureboot() {
     info "Extracting necessary swi content"
-    unzip -oq "$swipath" platform/firsttime .imagehash -d "$image_path"
+    # NOTE: boot/ is not used by the boot process but only extracted for kdump
+    unzip -oq "$swipath" 'boot/*' platform/firsttime .imagehash -d "$image_path"
```

Comment:
What is this change intended for? Is it relevant to this PR's context of fixing duplicate options?

Reply:
It's not related to this PR, I will remove.